deploy-manage/monitor/logging-configuration/logfile-audit-output.md

```diff
@@ -8,17 +9,19 @@ applies:
   stack: all
 ---
 
-% evaluate the applies section
+# Elasticsearch logfile audit output [audit-log-output]
 
-# Logfile audit output [audit-log-output]
+The `logfile` audit output is the only output for auditing. By default, it writes data to the `<clustername>_audit.json` file in the logs directory. The file is also rotated and archived daily or upon reaching the 1GB file size limit.
 
-The `logfile` audit output is the only output for auditing. It writes data to the `<clustername>_audit.json` file in the logs directory.
+In self-managed clusters, you can configure how the `logfile` is written in the `log4j2.properties` file located in `ES_PATH_CONF` (or check out the relevant portion of the [log4j2.properties in the sources](https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/core/src/main/config/log4j2.properties)). However, **Elastic strongly recommends using the default Log4j2 configuration**.
+
+Orchestrated deployments (Elastic Cloud, Elastic Cloud Enterprise (ECE), and Elastic Cloud on Kubernetes (ECK)) do not support changes in `log4j2.properties` files of the {{es}} instances.
```

Comment:
in this tight space we can just use acronyms. ideally, we wouldn't even use brackets and could just link to a definition

be careful about conflating "elastic cloud" with "elastic cloud hosted" 

```suggestion
Orchestrated deployments (ECH, ECE, and ECK) do not support changes in `log4j2.properties` files of the {{es}} instances.
```

alt:

```suggestion
No other deployment types support changes in `log4j2.properties` files of the {{es}} instances.
```